app/routes.py

```diff
@@ -1,2 +1,239 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response,request
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from datetime import datetime 
+#what comes from the "from" is the package name
+#what comes after the import is a variable that matches the name of thing we want to import
+tasks_bp = Blueprint("tasks", __name__,url_prefix=("/tasks"))
+
+goals_bp = Blueprint("goals", __name__, url_prefix=("/goals"))
+
+@goals_bp.route("",methods=["POST", "GET"])
+def handle_goal():
```

Comment:
Similar feedback about splitting these functions, and moving validation and dictionary-handling logic around that I made for `Task` below will also apply to `Goal` here. These are common themes for any model with CRUD operations.